Version1 adding .gitignore and deleting ViewModels and Models folder

alex308248 opened the pull request:
Trying to fix issue #2 and #3. Deleted the extra folder and add a raw .gitignore file.

Commit:
Remove files in ViewModels folder & Models folder

Commit:
Add a .gitignore file and remove the revelented files from branch

YLee2121 commented on .gitignore:
```diff
@@ -0,0 +1 @@
/*.gcno
**/xcshareddata/WorkspaceSettings.xcsettings

# End of https://www.toptal.com/developers/gitignore/api/macos,xcode,swift
```
Is it a .gitignore but compressed into one line?? 
Will it be a normal file after PR to the repo??

alex308248 replied:
Fixed. It happened due to the difference of newline character in Windows and Mac system.

YLee2121 replied:
OK Done reviewing!!!!
Nice Work!

YLee2121 reviewed (approved):
Done Review!

YLee2121 reviewed (approved):
Done Review!

YLee2121 committed:
MainView first PR (#5)

* construct five main tab

* MainView setup, Add intro View for five tabs

* Update the MainView:
	- create a enum Tab
	- separate the tabview into a independent view and add the tag

* organized the file again, it just happends

* documentation on MainView

* documentation on MainView

Embed all the tab into a NavigationView

---------

Co-authored-by: Alex Chao <alex308248@gmail.com>
